website/docs/developer/development-guide/code/integration-tests.md

```diff
@@ -8,6 +8,43 @@ While [unit tests](./unit-tests.md) cover individual functions, this does not co
 Consideration should be given to how the components of a large system interact, and how the layers fit together.
 These tests run realistic scenarios against a full system via scripts run in Bamboo.
 
+- [File Structure](#structure)
```

Comment:
I opted to keep our documentation centralized in docusaurus. If it's felt that I overdid it, then I can move/copy some information into the readme.